Documentation/Usage.md

```diff
@@ -67,7 +67,7 @@ get started:
 This creates the directory structure needed for executable targets. Any target
 can be turned into a executable target if there is a `main.swift` present in
 its sources. Complete reference for layout is
-[here](Reference.md#target-format-reference).
+[here](PackageDescriptionV3.md#target-format-reference).
```

Comment:
Sorry for the late review. I think we should reference V4 format now.

Reply:
Oh, sorry, didn't see the review. On it, will update it shortly.